python/packages/ai/teams/ai/planner/openai_planner.py

```diff
@@ -159,9 +159,11 @@ async def _complete_text(self, prompt_template: PromptTemplate):
             prompt_template_config.completion
         )
 
-        text_completion_client = self._sk.get_ai_service(TextCompletionClientBase)(self)
+        text_completion_client = OpenAITextCompletion(
```

Comment:
The `AzureOpenAIPlanner` may not work (I'm not sure), because it inherits `OpenAIPlanner`, and shared the logic here. And even it works, we may not expect SK ensures OpenAITextCompletion always work in the future.

Reply:
When running this code through real testing using the `ListBot`, this was throwing saying it could not find the service `TextCompletionClientBase` and that we were calling the method with too many params. I think something in `SK` isn't working for this method so I changed it to just use the class you are trying to get directly so that we can keep moving forward.

Reply:
See my comment at https://github.com/microsoft/teams-ai/pull/441#discussion_r1299504736. I would prefer to keep the implementation in my PR because that's how Semantic Kernel handles different (OpenAI & AzureOpenAI) client. Only use OpenAIClient here may cause potential issues when SK releases new package.